node_lambnik/scripts/update

```diff
@@ -21,9 +21,6 @@ then
 
         # Launch database container so we can populate it.
         docker-compose up -d database
-
-        # Load data fixtures.
-        ./scripts/updatedb
-
+        docker-compose exec database bash -c "./updatedb"
```

Comment:
There seems to be some race condition here that isn't accounted for in the updatedb script which checks the databases availability.  If I've previously built the containers via `docker-compose up` or `./scripts/server`, this script is successful.  However, after a `down` & `clean`, running `update` won't load the data.

Reply:
I for the life of me cannot seem to replicate this problem (not to say I doubt it's an issue). @KlaasH , does this happen on your machine as well?

Reply:
I think this is tangential enough, and with a workaround, to move forward.  Can you create an issue to follow up separately?

Reply:
Sure thing.